web/src/pages/api/v1/verify/[app_id].ts

```diff
@@ -39,6 +39,16 @@ const schema = yup.object({
         ),
     }),
   credential_type: yup.string().oneOf(Object.values(CredentialType)),
+  max_age: yup
+    .number()
+    .integer()
+    .min(3600, "Maximum root age cannot be less than 3600 seconds (1 hour).")
+    .max(
+      604800,
+      "Maximum root age cannot be more than 604800 seconds (7 days)."
+    )
+    .strict()
+    .optional(),
```

Comment:
Why do we want this to be optional? Shouldn't everyone be using an implementation that has a root age restriction

Reply:
roots are already limited to one week of validity at the on-chain/sequencer level, and we don't want to break existing integrations